src/sentry/api/bases/organization_events.py

```diff
@@ -162,15 +163,16 @@ def _get_terminal_event_id(self, direction, snuba_args, event):
         conditions = snuba_args["conditions"][:]
         conditions.extend(time_condition)
 
-        result = eventstore.get_events(
+        result = snuba.dataset_query(
```

Comment:
The current `get_events` implementation requires the `group_id` column which doesn't exist on transactions.